src/lib/commands/config.ts

```diff
@@ -144,6 +145,7 @@ export const registerCommands = (context: ExtensionContext) => {
                 );
               }
               await trackEvent(TRACKED_EVENTS.DELETE_ACCOUNT);
+              commands.executeCommand('hubspot.remoteFs.refresh');
```

Comment:
Executes a refresh when the default portal changes (above) or is deleted (below)